torrt/rpc/transmission.py

```diff
@@ -138,6 +154,14 @@ def method_get_version(self) -> str:
         result = self.query(self.build_request_payload('session-get', ['rpc-version-minimum']))
         return result['rpc-version-minimum']
 
+    @staticmethod
+    def __get_unwanted_files(torrent_info):
```

Comment:
Don't know whether we should mangle that method.

Reply:
Inlined.